src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -71,6 +80,21 @@ export const Chatbox: React.FC<ChatboxProps> = ({
     reset();
   });
 
+  const {
+    isShown: isEmojiPickerVisible,
+    hide: hideEmojiPicker,
+    toggle: toggleEmojiPicker,
+  } = useShowHide();
+
+  const addEmoji = (emoji: CustomEmoji) => {
+    if (emoji.colons) {
+      const message = getValues("message");
+      setValue("message", message + emoji.colons);
+    }
+    
+    hideEmojiPicker();
+  };
```

Comment:
See previous comment thread in https://github.com/sparkletown/sparkle/pull/1383/files#r640502523

Reply:
also no longer here as noted [here](https://github.com/sparkletown/sparkle/pull/1383#discussion_r645667869)

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1383/commits/38a695e20939ee86a1d519be133b9c6bd00e0101